### DISCUSSION TITLE
How to update the Hubs UI manually on AWS Server

### DISCUSSION THREAD

**Discussion by ilovebioz** [mozilla/hubs#5358] [Hubs Cloud Questions]
Hi, We are running a Mozilla hub cloud on AWS. We modified some things on the Hub client code, we would like to rebuild and update it directly to AWS. Can you show us the part we should override?

**Comment by msalafia** [comment]
I think you should go straight with `npm run deploy`.

**Comment by ilovebioz** [comment]
hi, when we use npm run deploy, we can succeed to distribute our update but we got 2 problems:
1. after deploying the admin page missing
2. the new hub client refused to  call external API
Do you have any experience with these?
